tests/tck/features/insert/Insert.IntVid.feature

```diff
@@ -1,445 +0,0 @@
-      | 'Tom'          | 80                             | 'Kitty'        |
-      | 'Tom'          | 80                             | 'Peter'        |
-      | 'Tom'          | 80                             | 'Lucy'         |
-    # get result through go
-    When executing query:
-      """
-      GO FROM hash("Lucy") OVER schoolmateWithDefault
-      YIELD schoolmateWithDefault.likeness,
-      $$.personWithDefault.name,
-      $$.personWithDefault.birthday,
-      $$.personWithDefault.department,
-      $$.studentWithDefault.grade,
-      $$.studentWithDefault.number
-      """
-    Then the result should be, in any order:
-      | schoolmateWithDefault.likeness | $$.personWithDefault.name | $$.personWithDefault.birthday | $$.personWithDefault.department | $$.studentWithDefault.grade | $$.studentWithDefault.number |
-      | 80                             | 'Laura'                   | 1578650400                    | 'engineering'                   | 'one'                       | 20190901008                  |
-      | 80                             | 'Amber'                   | 1578650400                    | 'engineering'                   | 'one'                       | 20180901003                  |
-    # insert multi version vertex
-    When executing query:
-      """
-      INSERT VERTEX person(name, age) VALUES hash("Tony"):("Tony", 18), hash("Mack"):("Mack", 19);
-      INSERT VERTEX person(name, age) VALUES hash("Mack"):("Mack", 20);
-      INSERT VERTEX person(name, age) VALUES hash("Mack"):("Mack", 21)
-      """
-    Then the execution should be successful
-    # insert multi version edge
-    When executing query:
-      """
-      INSERT EDGE schoolmate(likeness, nickname) VALUES hash("Tony")->hash("Mack")@1:(1, "");
-      INSERT EDGE schoolmate(likeness, nickname) VALUES hash("Tony")->hash("Mack")@1:(2, "");
-      INSERT EDGE schoolmate(likeness, nickname) VALUES hash("Tony")->hash("Mack")@1:(3, "");
-      """
-    Then the execution should be successful
-    # get multi version result through go
-    When executing query:
-      """
-      GO FROM hash("Tony") OVER schoolmate YIELD $$.person.name, $$.person.age, schoolmate.likeness
-      """
-    Then the result should be, in any order:
-      | $$.person.name | $$.person.age | schoolmate.likeness |
-      | 'Mack'         | 21            | 3                   |
-    # insert multi version edge
-    # When executing query:
-    # """
-    # INSERT VERTEX person(name, age) VALUES uuid("Tony"):("Tony", 18), uuid("Mack"):("Mack", 19);
-    # INSERT VERTEX person(name, age) VALUES uuid("Mack"):("Mack", 20);
-    # INSERT VERTEX person(name, age) VALUES uuid("Mack"):("Mack", 21)
-    # """
-    # Then the execution should be successful
-    # insert multi version edge with uuid
-    # When executing query:
-    # """
-    # INSERT EDGE schoolmate(likeness, nickname) VALUES uuid("Tony")->uuid("Mack")@1:(1, "");
```

Comment:
why delete it? the uuid

Reply:
My mistakes. I thought these were same feature files except vid type, not notice the uuid tests. Thanks. Have restored them.